src/bbc_server/config.ini

```diff
@@ -0,0 +1,3 @@
+[server]
+HOST = 
+PORT = 65432
```

Comment:
I don't think it is good to publish our port since we also want our users to decide on their own where to host their servers etc.

I think we should just leave this out.

Reply:
I deliberately placed the ports in a config file to allow for configuration. Users can still adjust this easily. If I wanted to 'hardcode' the port I woul've left it in the source code as it was before.
Having a default port makes the app more accessible to users in my opinion and reduces the likelihood of new users entering a commonly used port.